Assets/Scripts/Card.cs

```diff
@@ -97,6 +97,12 @@ public CardColor GetColor()
         return CardColor.NONE;
     }
 
+    //I had to add this for the hand highlighting because the other one was only returning NONE
```

Comment:
It was probably only returning NONE because the cards are all set to be hidden except for the owner of the card and the logic wasn't checking if it has authority.